packages/connect-examples/webextension-mv3-sw/src/manifest.json

```diff
@@ -5,6 +5,6 @@
     "background": {
         "service_worker": "serviceWorker.js"
     },
-    "permissions": ["tabs", "notifications", "scripting", "activeTab"],
```

Comment:
ok, are you sure that webextension does not need this in order to open a new tab with trezor connect popup? I assume that you tested it :D just trying to figure out why we had this here

Reply:
Yes, it works the whole flow without those permissions. Only `scripting` is required.